vpn_module.sh

```diff
@@ -37,10 +37,11 @@ VPN_RELAY_SET_LOCATION="mullvad relay set location"
 
 ## [Set VPN status parsing]
 # The first command cuts the status, which is compared to keywords below.
-# TODO: Add community submissions for other VPNs to make this section robust!
-VPN_STATUS="$($VPN_GET_STATUS | cut -d' ' -f3)"	# returns Connected/Connecting/<other>
-CONNECTED=Connected
-CONNECTING=Connecting
+# Note from Julia: this should cover most VPNs, if it's missing something let me know
+VPN_STATUS="$($VPN_GET_STATUS | grep -Eio 'connected|connecting|disconnected' \
+	| tr '[:upper:]' '[:lower:]')"
+CONNECTED=connected
+CONNECTING=connecting
```

Comment:
run vpn status command, look for keywords and translate it to lower case. the translation is so vpn providers using {CONNECTED,Connected,connected} will all register the same